### PR TITLE
deploy app on remote (#256)

### LINKED ISSUE #256: Write github actions similar to the ones on backend which automatically deploy these files on every commit in master

### PR DESCRIPTION
**Pull request by ostatni5**
close #256

### CODE REVIEW

File: .github/workflows/node.js.yml [hunk 1]
```diff
@@ -14,46 +14,108 @@ env:
...
+    env:
+      FRONTEND_HOST: 149.156.182.181
+      FRONTEND_USER: ubuntu
+
     steps:
-      - name: Checkout 🛎️
+      # get the repo to the github agent
+      - name: Checkout
         uses: actions/checkout@v2
+        with:
```
Comment (grzanka) [inline .github/workflows/node.js.yml:71]: Do we need submodules ? I haven't seen any in `ui` repo

File: .github/workflows/node.js.yml [hunk 2]
```diff
@@ -14,46 +14,108 @@ env:
...
+        # if: github.ref == 'refs/heads/master'
+        uses: appleboy/ssh-action@master
+        with:
+          host: ${{ env.FRONTEND_HOST }}
+          username: ${{ env.FRONTEND_USER }}
+          key: ${{ secrets.DEPLOY_C3_TEST }}
+          command_timeout: 30m
+          script: |
+            cd ui
+            npm install
```
Comment (grzanka) [inline .github/workflows/node.js.yml:119]: Which version of node we need at the frontend machine ?

image

Comment (grzanka) [reply .github/workflows/node.js.yml:117]: Consider replacing this with Docker one-liner

Comment (ostatni5) [reply .github/workflows/node.js.yml:116]: Node and nmp updated 
obraz

Moved to issue https://github.com/yaptide/ui/issues/264

**Review by grzanka — APPROVED** [review]
Looks good, lets test it

**Pull request merged**